types/src/connectors/notion.ts

```diff
@@ -8,11 +8,11 @@ import * as t from "io-ts";
 export type PageObjectProperties = PageObjectResponse["properties"];
 export type PropertyKeys = keyof PageObjectProperties;
 export type PropertyTypes = PageObjectProperties[PropertyKeys]["type"];
-export type NotionGarbageCollectionMode = "always" | "auto" | "never";
+export type NotionGarbageCollectionMode = "always" | "never";
 
 export function getNotionWorkflowId(
   dataSourceInfo: { workspaceId: string; dataSourceName: string },
-  gargbageCollectionMode: NotionGarbageCollectionMode = "auto"
+  gargbageCollectionMode: NotionGarbageCollectionMode
```

Comment:
Can we use something else than `never` and `always` here, maybe a regular boolean, like `isGarbageCollectorWorkflow`?

Reply:
WHy ?

Reply:
Cause it's not idiomatic! When you see the call sites, you can't really tell how this parameter influences the  workflow id.

Reply:
I don't think idiomatic is the right word here, but agree it's not super explicit on call sites.

Reply:
Thank you 🙇 .